cli/utils/init.config.ts

```diff
@@ -10,21 +12,33 @@ export default function provideInitConfig(
   shell: typeof shelljs,
   filesystem: typeof fs,
   fortaKeystore: string,
-  configFilename: string
+  configFilename: string,
+  agentId: string,
+  getFortaConfig: GetFortaConfig
 ) {
   assertExists(shell, "shell");
   assertExists(filesystem, "filesystem");
   assertIsNonEmptyString(fortaKeystore, "fortaKeystore");
   assertIsNonEmptyString(configFilename, "configFilename");
 
   return async function initConfig() {
-    if (!filesystem.existsSync(join(fortaKeystore, configFilename))) {
+    const filePath = join(fortaKeystore, configFilename);
+    if (!filesystem.existsSync(filePath)) {
+      // Create file
       console.log(`Creating ${configFilename}...`);
       const copyConfigResult = shell.cp(
         join(__dirname, "..", "commands", "init", configFilename),
         fortaKeystore
       );
       assertShellResult(copyConfigResult, `Error creating ${configFilename}`);
+
+      const fortaCon = getFortaConfig();
+
+      // Save initial forta config
+      console.log(`Saving agentId: ${agentId} in ${configFilename}`);
+
+      const data = assign({agentId}, fortaCon);
+      fs.writeFileSync(filePath, stringify(data, undefined, 1))
```

Comment:
One weird thing about this json library is in order to serialize json with comments you have to pass in a non-null/empty value in the third param ([link to docs ](https://github.com/kaelzhang/node-comment-json#stringify)).